bika/lims/utils/analysisrequest.py

```diff
@@ -421,8 +422,12 @@ def create_retest(ar):
     renameAfterCreation(retest)
 
     # 2. Copy the analyses from the source
-    criteria = dict(full_objects=True, retracted=False, reflexed=False)
-    for an in ar.getAnalyses(**criteria):
+    intermediate_statuses = ['retracted', 'reflexed']
```

Comment:
better name it `intermediate_states`

Reply:
state, status, states...I never know what to choose..